packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.10
+    tag: v1.0.11-rc1
```

Comment:
Why do we need to bump the cis-operator version? Is there a commit added for this change to cis-operator?

Reply:
Yes the pull policy for the images are updated in the cis-operator code as well.
I see you already checked and approved that.. https://github.com/rancher/cis-operator/pull/171

Reply:
Have merged the PR and created the relevant tag. Please approve.